### PR TITLE
Add height weight validations

### PR DESCRIPTION
**Pull request by hamza-vd**
See this [issue](https://github.com/opensrp/opensrp-client-unicef-tunisia/issues/149)

### CODE REVIEW

File: opensrp-growth-monitoring/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -50,5 +50,6 @@
     <string name="catchment_area">Catchment Area</string>
     <string name="user_image">User Image</string>
     <string name="weight_is_required">The patient weight cannot be empty.</string>
+    <string name="height_is_required">"Child height should not be less than 0cm and not above 100 cms."</string>
```
Comment (qiarie) [inline opensrp-growth-monitoring/src/main/res/values/strings.xml:53]: Remove the quotes from the string definition

Comment (hamza-vd) [reply opensrp-growth-monitoring/src/main/res/values/strings.xml]: Removed

File: opensrp-growth-monitoring/src/main/java/org/smartregister/growthmonitoring/fragment/RecordGrowthDialogFragment.java [hunk 2]
```diff
@@ -290,20 +291,23 @@ private void saveGrowthRecord(boolean isToday) {
...
-
-        Float weight = Float.valueOf(weightString);
-        weightWrapper.setWeight(weight);
-
-        if (monitorGrowth) {
             String heightString = editHeight.getText().toString();
             if (!heightString.isEmpty()) {
-                Float height = Float.valueOf(heightString);
+                float height = Float.parseFloat(heightString);
+                if (height <= 0f || height > 1000f) {
```
Comment (dubdabasoduba) [inline opensrp-growth-monitoring/src/main/java/org/smartregister/growthmonitoring/fragment/RecordGrowthDialogFragment.java:297]: Is this 1000 or 100

**Review by dubdabasoduba — APPROVED** [review]

**Review by dubdabasoduba — CHANGES_REQUESTED** [review]
Please check my comment earlier

File: opensrp-growth-monitoring/src/main/java/org/smartregister/growthmonitoring/fragment/RecordGrowthDialogFragment.java (hunk 2 above)
Comment (hamza-vd) [reply opensrp-growth-monitoring/src/main/java/org/smartregister/growthmonitoring/fragment/RecordGrowthDialogFragment.java:297]: Let me correct, it.

Comment (hamza-vd) [reply opensrp-growth-monitoring/src/main/java/org/smartregister/growthmonitoring/fragment/RecordGrowthDialogFragment.java]: Its fixed now.

**Review by qiarie — APPROVED** [review]

**Review by dubdabasoduba — APPROVED** [review]

**Pull request merged**